gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageClientImpl.java

```diff
@@ -186,10 +191,21 @@ private long getWriteGeneration(StorageResourceId resourceId, boolean overwrite)
   }
 
   private static Storage createStorage(
-      Credentials credentials, GoogleCloudStorageOptions storageOptions) {
+      Credentials credentials,
+      GoogleCloudStorageOptions storageOptions,
+      List<ClientInterceptor> interceptorsList) {
```

Comment:
nit: interceptorsList -> interceptors (having type name is variable name is a code smell)